examples/democoin/mock/validator.go

```diff
@@ -38,6 +38,11 @@ func (v Validator) GetDelegatorShares() sdk.Rat {
 	return sdk.ZeroRat()
 }
 
+// Implements sdk.Validator
```

Comment:
Is this sdk.Validator, or `x/stake/types.Validator`?`

Reply:
It is `sdk.Validator` (`types/stake.go`), to which `GetRevoked()` was added.